app/src/main/res/xml-v25/shortcuts.xml

```diff
@@ -0,0 +1,15 @@
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+    <shortcut
+        android:shortcutId="mySession"
+        android:enabled="true"
+        android:icon="@drawable/ic_heart_orange_24dp"
+        android:shortcutShortLabel="@string/favorite_title"
+        android:shortcutLongLabel="@string/favorite_title">
+        <intent
+            android:action="android.intent.action.VIEW"
+            android:targetPackage="io.github.droidkaigi.confsched2018.debug"
+            android:targetClass="io.github.droidkaigi.confsched2018.presentation.MainActivity">
```

Comment:
`targetPackage` = `application ID`. But application ID will change by build type.
I implemented `shortcuts.xml` 2 files for build type.
If you know best practice , please tell me 🙇

Reply:
I using it but same..
I don't have that best practice 🙇